pkg/image/apis/image/helper.go

```diff
@@ -427,6 +427,39 @@ func ImageConfigMatchesImage(image *Image, imageConfig []byte) (bool, error) {
 	return v.Verified(), nil
 }
 
+// ReorderImageLayers mutates the given image. It reorders the layers in ascending order.
+// The layers are present in the schema v1 and v2 in ascending order.
```

Comment:
Ascending order matches the order of layers in schema 2. Schema 1 has reversed (descending) order of layers.